src/pages/billing/Project.js

```diff
@@ -51,40 +49,36 @@ export const ProjectUsersList = ajaxCaller(class ProjectUsersList extends Compon
   }
 
   render() {
-    const { projectName, ajax: { Billing } } = this.props
+    const { project: { projectName, creationStatus }, ajax: { Billing } } = this.props
     const { projectUsers, loading, updating, filter, addingUser, deletingUser, editingUser } = this.state
     const adminCanEdit = _.filter(({ roles }) => _.includes('Owner', roles), projectUsers).length > 1
 
     return h(Fragment, [
-      h(TopBar, { title: 'Billing', href: Nav.getLink('billing') }, [
-        search({
-          wrapperProps: { style: { marginLeft: '2rem', flexGrow: 1, maxWidth: 500 } },
-          inputProps: {
-            placeholder: 'SEARCH USERS',
-            onChange: e => this.setState({ filter: e.target.value }),
-            value: filter
+      div({ style: { padding: '1.5rem 3rem' } }, [
```

Comment:
```suggestion
      div({ style: { padding: '1.5rem 3rem', flexGrow: 1 } }, [
```
together with the suggestion below, this should fix the double scroll issue.